tests/test_oidc_auth.py

```diff
@@ -140,18 +140,17 @@ def construct_jwt(
     msg: Dict[str, Any] = None,
     headers: Optional[Dict] = None,
 ):
-    iat = datetime.datetime.utcnow()
-    exp = datetime.datetime.utcnow() + datetime.timedelta(
-        days=1000000
-    )  # This or patch jwt.decode
+    iat_timestamp = datetime.datetime.utcnow().timestamp()
+    delta_day = 1000000   # Unit: day
```

Comment:
I'm very much ok, with the change, but could you please add a comment in your commit message why it was necessary and/or why windows can´t handle the timedelta (or at least what was off with the old way)?

Reply:
![image](https://user-images.githubusercontent.com/26598493/124351798-4e792f00-dbf4-11eb-8de5-065e38cb1873.png)
There will have OSError for int(exp.timestamp()) in windows 10. you can test it.

Reply:
I'm not currently running a win10 system, so I kinda rely on what you tell me. Also I'm quite curious why it it doesn't work but can't check myself :smirk: 

By the way could you fix the naming there (plural) and remove the redundant comment (the var already suggests that it is the unit days)?

Reply:
Done